src/sagemaker/jumpstart/cache.py

```diff
@@ -242,6 +291,26 @@ def _get_manifest_key_from_model_id_semantic_version(
 
         raise KeyError(error_msg)
 
+    def _get_open_source_manifest_key_from_model_id(
+        self,
+        key: JumpStartVersionedModelId,
+        value: Optional[JumpStartVersionedModelId],  # pylint: disable=W0613
+    ) -> JumpStartVersionedModelId:
+        """Get open source manifest key from model id."""
```

Comment:
non-descriptive docstring. Since the function is internal, you can just remove the docstring I reckon.